Source/ACE.Server/WorldObjects/Player_Xp.cs

```diff
@@ -4,19 +4,26 @@
 using ACE.DatLoader;
 using ACE.Entity.Enum;
 using ACE.Entity.Enum.Properties;
+using ACE.Server.Managers;
 using ACE.Server.Network.GameMessages.Messages;
 using ACE.Server.WorldObjects.Entity;
 
 namespace ACE.Server.WorldObjects
 {
     partial class Player
     {
+
+        private static double XpModifier { get { return PropertyManager.GetDouble("xp_modifier"); } }
```

Comment:
I wouldn't use properties here.

XpModifier really isn't a property of Player.

Instead, I would either use a local variable when you want to use a property, or, just reference the property directly.

So, it would be something like this:
xpModifier = PropertyManager.GetDouble("xp_modifier");
xpToAdd = incomingXp * xpModifier;

- or -

xpToAdd = incomingXp * PropertyManager.GetDouble("xp_modifier");